Felienne/hedy
# [UI] Improves language selector

**Description**
In this PR we clean-up the language selector due to the increasing amount of languages. The current dropdown menu is getting too large and even grows out-of-view. In this PR we fix this issue by splitting the dropdown menu into two columns instead of one. We also re-write the structure so we are able to easily adjust this to 3 or 4 columns in the future.

**Fixes**
This PR fixes #2310

**How to test**
Hover the language selector and verify that it is now two columns instead of one. Also verify that the search is still working and the languages automatically re-position (!) on searching. Making sure that it is always an evenly divided view.

**New UI**
![Schermafbeelding 2022-03-25 om 13 21 04](https://user-images.githubusercontent.com/38155424/160120129-4328df19-cc52-4730-92d5-1762ff69112c.png)
